examples/compose.js

```diff
@@ -0,0 +1,53 @@
+import merge from 'lodash/object/merge';
+import map from 'lodash/collection/map';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return map(composables, 'compose.' + descriptorName);
+};
+
+const createStamp = descriptor => {
+  return ({ instance } = {}) => {
+    const proto = {};
+    const obj = instance || {};
+
+    // Set up prototype
+    Object.assign(proto, descriptor.methods);
+    Object.setPrototypeOf(obj, proto);
```

Comment:
Yeah, I'm using `Object.create()` where it's possible, but if a user passes an instance in to be mutated, we can't just `Object.create()` a new instance. For example, a user might want to use an array or a function, and `Object.create()` won't work for those cases.